### DISCUSSION TITLE
Where are all of my songs?

### DISCUSSION THREAD

**Discussion by VirtualComputersVM** [KRTirtho/spotube#1777] [Q&A]
Signed in with Spotify, yet my playlist is displaying less songs than it contains. Playlist contains 411 songs on Spotify but Spotube is only displaying 59 Songs. Is this because of the songs not being on any of the audio sources?

**Comment by KRTirtho** [comment]
Nope, may be a pagination issue. You can scroll down and it should be loaded gradually.

**Reply by KRTirtho** [reply]
Or, if you just play the playlist all the tracks will be played at once. It might take a few seconds to start

**Reply by jmoney1999** [reply]
this doesnt seem to work, played some music from the playlist and didnt load anything new the song ends and thats all. is there something im missing on the android version?
